bundler/index.js

```diff
@@ -4,70 +4,115 @@ const HtmlParser = require('./modules/htmlParser');
 
 class Bundler {
   constructor(
-    fileManager = new FileManager(),
-    uglifier = new Uglifier(),
-    htmlParser = new HtmlParser()
+    pathConfig,
+    tagConfig,
   ) {
-    this.fileManager = fileManager;
-    this.uglifier = uglifier;
-    this.htmlParser = htmlParser;
+    this.pathConfig = pathConfig;
+    this.tagConfig = tagConfig;
+    this.fileManager = new FileManager();
+    this.uglifier = new Uglifier();
+    this.htmlParser = new HtmlParser(tagConfig);
```

Comment:
Let's left `new FileManager()`, `new Uglifier()` and `new HtmlParser(tagConfig)` as dependencies.

The reason why I suggest this is to be able to mock them clear and easy in our future tests.